src/managers/auth.py

```diff
@@ -8,13 +8,11 @@
 import re
 import subprocess
 from dataclasses import asdict, dataclass
-from typing import TYPE_CHECKING, Optional, Set
 
-from literals import REL_NAME
-from snap import KafkaSnap
+from ops.pebble import ExecError
 
-if TYPE_CHECKING:
-    from charm import KafkaCharm
+from core.cluster import ClusterState
+from vm_workload import KafkaWorkload
```

Comment:
We should not import from `vm_workload` here. This breaks the clean structure of imports, as I also say [here](https://github.com/canonical/kafka-operator/pull/161/files#diff-9ca1291973b66502ad17535794d685ef375646a4b1ad69129c40324012eb6119R28)

Reply:
Note that I have noticed that this in general seems to happen on all managers. We should definitely change that. It should be rather easy (it is done like this in Zookeeper), but I believe it makes a lot of difference to have a clear import structure